RELEASES.md

```diff
@@ -1,3 +1,122 @@
+Version 1.43.0 (2020-04-23)
+==========================
+
+Language
+--------
+- [Fixed using binary operations with `&{number}` (e.g. `&1.0`) not having
+  the correctly inferred type.][68129]
```

Comment:
Did you mean
```suggestion
  the correct inferred type.][68129]
```

Reply:
I'm pretty sure both are correct.

Reply:
I'm not fluent (I'm French) but this seems odd to me. What about:
```suggestion
  the type inferred correctly.][68129]
```

Reply:
@LeSeulArtichaut I think all three versions are fine.

In "correct inferred type," both "correct" and "inferred" are adjectives modifying "type"; i.e., it is saying that the final type is correct, and, separately, that the type itself was inferred. In "correctly inferred type", "correct" is an adverb modifying "inferred", i.e., it indicates that the inference operation itself was performed correctly.